blueprints/ember-composable-helpers/index.js

```diff
@@ -1,7 +1,9 @@
 module.exports = {
   normalizeEntityName: function() {},
 
-  afterInstall: function() {
-    return this.addBowerPackageToProject('ember-cli-shims', '~0.1.1');
+  afterInstall: function(options) {
+    if (!('ember-cli-shims' in options.project.addonPackages)) {
```

Comment:
`addonPackages` is an object with each package as a key and I think the `in` operator for a key of an object works all the way back to ES3. I'm willing to be wrong about that though - just let me know.